### DISCUSSION TITLE
Parallel environment on Apple M1 pro

### DISCUSSION THREAD

**Discussion by Pteravela** [nextfoam/baram#85] [Q&A]
Hello!

First, thank you so much for this software which allows us to exploit the power of Apple chips!

However, I can't run AhmedBody on parallel environment. I tried 2, 3,.... to 8 cores

It run very well on 1 core. 

I have a macbook pro M1 pro 16 Go
I run Baram 24.1.2
mpirun (Open MPI) 5.0.3
[stderr.log](https://github.com/user-attachments/files/15804440/stderr.log)

Do you have any idea?

Thank you!!

Rémi

**Comment by jiban** [comment]
Hi Rémi.
It's my pleasure for you to have interest in BARAM.
And I'm sorry for the inconvenience.
v24.1.2 was built with open-mpi 4.x, and I found that it had compatibility issue with the latest open-mpi package of 5.0.3.
I've uploaded updated version of BARAM package for macOS just now.
Would you please try with this version?
You can download it again on [baramcfd.org](https://baramcfd.org/).
Image file name would be `BARAM-24.2.0-patch.dmg`.
Thank you.

-- Jake

**Comment by Pteravela** [comment]
Hi Jake,

It works! Thank you so much!

Rémi.
